### ISSUE TITLE
[Enhancement] List already downloaded backups

### ISSUE THREAD

**Issue by yeawellwhatever** [evertonstz/pynps#9]
NPS Browser has this feature, which makes it easier to track backups I have already. I believe it records all downloaded backups, DLC and updates under npsSettings.dat .

**Comment by evertonstz** [comment]
I've not used this

**Comment by evertonstz** [comment]
Good idea! I'll take a look into implementing a history database, it's useful. Its been a while since I used NPS, but it just only checks their database, It doesn't look for files in the download folder, right? Like a downloaded game that you've fully downloaded, transfered to Vita and deleted from PC will still be listed in the history.

**Comment by yeawellwhatever** [comment]
yup, it doesnt look inside the downloaded folder. I tested by renaming the folders and relaunching NPS. 

Although it wont be a bad idea to implement based on the folders either. At least its easier to "mark" as downloaded manually.
